picard/formats/mp4.py

```diff
@@ -182,7 +192,13 @@ def _save(self, filename, metadata):
             elif name in self.__r_int_tags:
                 try:
                     tags[self.__r_int_tags[name]] = [int(value) for value in values]
-                except ValueError:
+                except (ValueError, TypeError) as e:
+                    log.error("MP4: Cannot save tag %s from %s as %r in %r: %s",
```

Comment:
The only other call to `log.error` in this file doesn't have an `MP4` prefix in the message.

Reply:
Will add to other call.